packages/invoices/src/OutboundShipment/ListView/Toolbar.tsx

```diff
@@ -18,7 +18,7 @@ export const Toolbar: FC<{
   const onDelete = useDeleteSelectedOutbounds();
 
   const key = 'comment' as keyof OutboundRowFragment;
-  const filterString = filter.filterBy?.[key]?.like as string;
+  const filterString = (filter.filterBy?.[key]?.like as string) || '';
```

Comment:
This just prevents this React error:
<img width="1070" alt="Screen Shot 2022-03-15 at 10 31 50 AM" src="https://user-images.githubusercontent.com/5456533/158291120-2353381f-58f4-4204-b497-70a53e68eb70.png">

Reply:
Oh yes, very annoying. I cause this mui error a lot 😁